src/controllers/kycController.ts

```
@@ -0,0 +1,335 @@
+  next: NextFunction,
+): Promise<void> {
+  try {
+    const userId = req.apiKey?.userId;
+    if (!userId) throw new AppError("User-scoped API key required", 401);
+
+    const body = requestUploadUrlSchema.parse(req.body);
+
+    // Generate a document ID if not provided
+    const documentId =
+      body.document_id ??
+      crypto.randomUUID();
```

ESLint

[error] Replace `⏎······body.document_id·??⏎·····` with `·body.document_id·??`

(prettier/prettier)